src/librustc_codegen_ssa/back/link.rs

```diff
@@ -1241,6 +1238,9 @@ fn add_late_link_args(
             cmd.args(args);
         }
     }
+    if let Some(args) = sess.target.target.options.late_link_args.get(&flavor) {
+        cmd.args(args);
+    }
```

Comment:
This is pretty fragile.
"More specific" options conventionally go after "less specific" options.
If some option lists in target specs share tails rather than starts and their order is important, then I guess they'll have to be written explicitly in target specs.

Reply:
🏳️